dotcom-rendering/src/components/AustralianTerritorySwitcher.importable.tsx

```diff
@@ -0,0 +1,84 @@
+import { css } from '@emotion/react';
+import { removeCookie, setCookie } from '@guardian/libs';
+import { space } from '@guardian/source-foundations';
+import {
+	Button,
+	ChoiceCard,
+	ChoiceCardGroup,
+	SvgChevronDownSingle,
+	SvgChevronUpSingle,
+} from '@guardian/source-react-components';
+import { isString } from 'lodash';
+import { useState } from 'react';
+import type { AustralianTerritory } from '../types/territory';
+
+const styles = css`
+	padding: ${space[2]}px;
+`;
+
+const territories = {
+	'AU-NSW': 'New South Wales',
+	'AU-QLD': 'Queensland',
+	'AU-VIC': 'Victoria',
+} as const satisfies Record<AustralianTerritory, string>;
+
+const controls = 'change-territory';
+
+type Props = {
+	targetedTerritory: AustralianTerritory;
+};
+export const AustralianTerritorySwitcher = ({ targetedTerritory }: Props) => {
```

Comment:
Curious what happens in a no-js experience - I don't think there's a lot we can do but perhaps worth having it written down in this PR.

Reply:
I guess we could make this a client-only island…

Reply:
Went for `onInteraction` in the end